params/config.go

```diff
@@ -1172,3 +1294,19 @@ func CheckETxChainID(ourID *big.Int, txID *big.Int) bool {
 	}
 	return false
 }
+
+// CurrentOntology is used to retrieve the MapContext of a given block.
+func (c *ChainConfig) CurrentOntology(number []*big.Int) []int {
+	forkNumber := number[0]
+
+	switch {
+	case forkNumber.Cmp(LovelaceBlock) >= 0:
```

Comment:
Cases are backwards for return. Go ahead and make only a single case here just to be safe.